src/sql/azdata.proposed.d.ts

```diff
@@ -385,6 +385,29 @@ declare module 'azdata' {
 		title: string;
 	}
 
+	export interface ConnectionProvider extends DataProvider {
+		/**
+		 * Changes the password for a SQL Server connection profile.
```

Comment:
probably shouldn't refer to "SQL Server" since presumably other providers could implement this method too?